### PR TITLE
Install rhoda and update other operator versions

### PR DESCRIPTION
**Pull request by rajivnathan**
- Installs the rhoda (DBaaS) operator which will automatically install the mongo, crunch and cockroach operators.
- Updates the starting versions for some of the other operators to speed up installation process
- Mention installing latest OCP 4.10.x in README

### CODE REVIEW

File: setup/README.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ This document describes how to use the setup tool to set up a Dev Sandbox enviro
 
 == Prereqs
 
-. Provision the *latest available* GA version of OCP on AWS with sufficient resources: 3 `m5.4xlarge` master nodes and 3 `m5.2xlarge` worker nodes.
+. Provision the *latest available* GA version of *OCP 4.10.x* on AWS with sufficient resources: 3 `m5.4xlarge` master nodes and 3 `m5.2xlarge` worker nodes.
```
Comment (alexeykazakov) [inline setup/README.adoc:7]: We had to resize our production clusters to m5.8large due to actual API server memory consumption. So I'm thinking maybe we should change the recommended size for the performance tests too...
WDYT?

File: setup/operators/installtemplates/rhoda.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    metadata:
+      name: ${DBAAS_NAMESPACE}
+  - apiVersion: operators.coreos.com/v1alpha1
+    kind: CatalogSource
+    metadata:
+      name: dbaas-operator-catalog
+      namespace: ${DBAAS_NAMESPACE}
+    spec:
+      sourceType: grpc
+      image: quay.io/tchughesiv/dbaas-operator-catalog:v0.1
```
Comment (alexeykazakov) [inline setup/operators/installtemplates/rhoda.yaml:17]: This is what we use in staging: https://github.com/codeready-toolchain/sandbox-sre/blob/master/resources/sandbox-stage.gb17.p1.openshiftapps.com/install/member/rhoda-operator/dbaas-operator.yaml#L13
Let's double check with the RHODA folks what image we should use.

Comment (rajivnathan) [reply setup/operators/installtemplates/rhoda.yaml:17]: Nvm, I was looking at an older version of the rhoda.yaml. I'll use the latest one.

File: setup/README.adoc (hunk 1 above)
Comment (rajivnathan) [reply setup/README.adoc:7]: Makes sense

File: setup/operators/installtemplates/rhoda.yaml (hunk 2 above)
Comment (rajivnathan) [reply setup/operators/installtemplates/rhoda.yaml]: Done in 4ab0c915cf1b96997e79fbe4c034ab5cfa134f00

File: setup/README.adoc (hunk 1 above)
Comment (rajivnathan) [reply setup/README.adoc]: Done in 74c3babc382cff841adb2468ac9bdf0f3be52808

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**